samples/sample-custom-subscription-hook/src/sample-custom-user-subscription-plugin-item/component.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { useEffect, useState } from 'react';
+
+import * as BbbPluginSdk from 'bigbluebutton-html-plugin-sdk';
+
+function SampleCustomSubscriptionPlugin():
+ React.ReactElement{
+  
+  const currentUser = BbbPluginSdk.useCurrentUser();
+
+  let selfName = ""
+  if (currentUser) {
+    selfName = currentUser.name
+  }
+
+  const dataResult = BbbPluginSdk.useCustomSubscription(`
+    subscription {
+      user_aggregate(where: {name: {_eq: "${selfName}"}}) {
+        aggregate {
+          count
+        }
+      }
+    }
+  `);
```

Comment:
Use variable instead of text (security).
Ensure the bigbluebutton-html5 uses the query + variables as key.